social/communication_and_social_stuff.md

```diff
@@ -0,0 +1,22 @@
+# Communication and Social Stuff
+## Email Lists/ Google Groups
+- saturn-dev
+- dsp-analysis-platform
+
+## Calendars
+- DSDE Events
+
+## Slack
+Slack is our primary communication method for questions, issues, and discussion, especially while we work remotely. For suggestions on channels to check out and how to join it, go to `tooling/Slack.md`.
+
+## Affinity Groups
+These groups all have Google groups you can join for email lists and calendars. Check them out [here](intranet.broadinstitute.org/broad-affinity-groups)!
+
+
+## Saturn Culture Things
+- **Terryl the Turtle**: they are our mascot and a wonderful stuff animal that goes on the desk of whoever the user liaison is
+- **Vester runs**: the team regularly goes to the best cafe in Kendall square, Vester, as a group. If you are going and are feeling socially inclined, feel free to ask around if anyone wants to come with you.
+
+## DSP Culture Things
+- **Broadiversary** : the anniversary that you started working at the Broad
+-  **cake table** : table in the kitchen where generally baked goods and candy for public consumption is placed. Typically populated by cakes celebrating Broadiversaries or candies from wherever a group of people just went on a business trip
```

Comment:
Nit: Looks like there is an extra space at the start of this line, and probably capitalize to `Cake Table`

Reply:
Nit: typo I think, 'stuff animal' should be 'stuffed animal'